commit 4af766ba41cc8c7e3261d2075c806bd7f185f92d
Author: Dave Anderson <anderson@redhat.com>
Date:   Wed Jul 1 15:30:15 2015 -0400

    Fix for the "timer" command when run on a kernel with a large number
    of cpus.  Without the patch, the command may fail prematurely with
    a dump of the internal crash utility allocated buffer statistics
    followed by the message "timer: cannot allocate any more memory!",
    (anderson@redhat.com)

diff --git a/kernel.c b/kernel.c
index cb8084a..dd1401b 100644
--- a/kernel.c
+++ b/kernel.c
@@ -7833,6 +7833,7 @@ new_timer_list_format:
                                 tdx++;
                         }
 		}
+		FREEBUF(timer_list);
 	}
 
 	FREEBUF(timer_list_buf);

